index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
```

Comment:
Consider using a descriptive title for your webpage. Providing unique, concise HTML page titles helps users with disabilities quickly understand a web page’s content and purpose. You can read more about this [here](https://accessibility.iu.edu/creating-content/web-content/titles.html)